quickstart_lists/README.md

```diff
@@ -0,0 +1,26 @@
+This directory contains 'starter lists' of packages for bootstrapping and syncing on-prem-builders.
+
+Background: Historically we bootstraped on-prem-builders by downloading all the packages in 'core'
+for all targets. That amounted to about 15GB, and was both too much and too little, in that many of
+the packages weren't needed, and for many patterns (effortless) other origins were needed.
+
+The basic naming pattern is TASK\_ARCH\_CHANNEL. This due to a limitation of the input file
```

Comment:
Change "basic naming pattern" to be "basic file name pattern" might be better as it was confusing what the naming pattern was referring to. 

Also, not sure we need the explanation of input file limitation as there's nothing actionable there for the reader.